crates/sui-indexer/src/test_utils.rs

```diff
@@ -81,7 +86,7 @@ pub async fn start_test_indexer_impl(
     // Reduce the connection pool size to 10 for testing
     // to prevent maxing out
     let pool_config = ConnectionPoolConfig {
-        pool_size: 5,
+        pool_size: 10,
```

Comment:
can we keep these at a lower number?

Reply:
I increased this b/c the conn pool is shared by both writers and readers now, also the comment said 10, I am okay if there is strong reason to keep it 5 though

Reply:
yeah lets keep it at 5, this just reduces the amount of resources a test will actually need